config/samples/applicationMonitoring.yaml

```diff
@@ -108,13 +108,13 @@ spec:
     #
     # Suggested starting point for resources below:
```

Comment:
I'd remove this comment, the description above already mentions that these values should be adjusted.
It's not really a suggestion anymore when it's there by default